WallTrailingGame/src/main/java/com/projecttango/examples/java/planefitting/ConfigActivity.java

```diff
@@ -118,6 +156,16 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         if(resultCode == RESULT_OK && requestCode == 10){
             uriSound = data.getData();
             contextSound = this;
+            if(mRewardSoundDist == 0.0)
+            {
+                mButton.setText("Play");
+            }
+            else
+            {
+                mButton.setText("Play");
+                mButton.setTextColor(0xFF00FF00);
```

Comment:
Can you put the "Activate Start Button" commands in a function?  The function could do 

1. Set the Play Text
2. Set Text Color to Green
3. Set Button Outline to Green

I suggest this because you use these three sets of commands in multiple places!